homeassistant/components/eway/strings.json

```
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Eway Inverter Setup",
+        "description": "Configure your Eway micro inverter connection",
+        "data": {
+          "mqtt_host": "MQTT Host",
+          "mqtt_port": "MQTT Port",
+          "mqtt_username": "MQTT Username",
+          "mqtt_password": "MQTT Password",
+          "device_id": "Device ID",
+          "device_sn": "Device Serial Number",
+          "device_model": "Device Model",
+          "scan_interval": "Scan Interval (seconds)"
```

Note that Home Assistant requires sentence-casing on all strings to clearly differentiate names from all words that can be translated:

```suggestion
        "title": "Eway inverter setup",
        "description": "Configure your Eway micro inverter connection",
        "data": {
          "mqtt_host": "MQTT host",
          "mqtt_port": "MQTT port",
          "mqtt_username": "MQTT username",
          "mqtt_password": "MQTT password",
          "device_id": "Device ID",
          "device_sn": "Device serial number",
          "device_model": "Device model",
          "scan_interval": "Scan interval (seconds)"
```